submissions/EmmanuelIdiaghe/javascript/index.js

```diff
@@ -24,4 +24,24 @@ var productList = [
 
 for (let loopNum=0; loopNum < productList.length; loopNum++) {
     console.log(productList[loopNum].productName + " with " + productList[loopNum].imageUrl + " is sold for " + productList[loopNum].productPrice);
+}
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+/*
+    Task: Print Numbers from 1 - 116 with conditionals
+*/
+
+const positiveDiff = (Math.abs(7 - 123));
+
+for (var counter = 1; counter <= positiveDiff; counter++) {
+    if(((counter % 3) == 0) && ((counter % 5) == 0)) {
```

Comment:
The brackets are too much. Doing this `(counter % 3 == 0 && counter % 5 == 0 )` will still work file I guess. Same for others too